qualification_detector_node:
  ros__parameters:
    orange_lower: [0, 20, 40]
    orange_upper: [35, 255, 255]
    min_area: 300
    publish_debug: true
    gate_width_meters: 1.5
    post_height_meters: 1.4
    smoothing_alpha: 0.4

qualification_navigator_node:
  ros__parameters:
    mission_depth: -0.8
    search_forward_speed: 0.4
    search_rotation_speed: 0.3
    approach_speed: 0.6
    approach_stop_distance: 3.0
    alignment_distance: 3.0
    alignment_threshold: 0.06 
    alignment_max_time: 20.0
    final_approach_speed: 0.5
    final_approach_threshold: 0.10
    commit_distance: 1.2
    commit_alignment_threshold: 0.10
    commit_speed: 0.8 
    passing_trigger_distance: 0.6
    passing_speed: 1.0
    uturn_forward_speed: 0.3
    uturn_angular_speed: 0.4
    uturn_depth: -0.8
    

    gate_x_position: 0.0
    auv_length: 0.46
    
    clearance_margin: 4.0         
    
    # Gate width
    gate_width: 1.5

    commit_travel_distance: 3.51